py-rse/automate.Rmd

```diff
@@ -234,11 +169,11 @@ $ ls -l -t data/moby_dick.txt results/moby_dick.csv
 As a further test:
 
 1.  Delete `results/moby_dick.csv` and run `make` again (case #1);
```

Comment:
Should these be added to the exercise section instead? I think in cases such as these, when files are created incrementally throughout the chapter, it is advantageous to have the exercises interspersed with the learning material instead of separated at the end. But unless there are several other lessons with the same layout as this one, I guess we should just stick them at the end for consistency?

Reply:
We decided this a while but I personally disagree with having all the exercises at the end, to me it feels it breaks the flow of the narrative and lessons... maybe we should bring this up again at the meeting? :man_shrugging: